### ISSUE TITLE
Lab environment not responding

### ISSUE THREAD

**Issue by SimonKrughoff** [pangeo-data/pangeo#254]
I'm giving http://pangeo.pydata.org/ a spin and it was working great over the weekend, but now I can't get the JupyterLab environment to respond.  I can authenticate with GitHub and login.  I see the Files pane and a launcher, but doing anything beyond that hangs.  For example, I never get a prompt if I launch a terminal.

FWIW, I'm working on the [Large Synoptic Survey Telescope](lsst.org) and we are building a science platform on the same technology stack.  See [this note](https://sqr-018.lsst.io) for some of the details.  It's very exciting to see others making such great headway in this space.

**Comment by mrocklin** [comment]
Hrm, can you verify that this is still the case?  When I log in to your session (I have admin access (FYI nothing here is secure)) things seem to respond normally for me.

![image](https://user-images.githubusercontent.com/306380/40020918-64321db0-5791-11e8-86ce-cb95d59ba753.png)

> FWIW, I'm working on the Large Synoptic Survey Telescope and we are building a science platform on the same technology stack. See this note for some of the details. It's very exciting to see others making such great headway in this space.

I'm glad to see involvement from the LSST!

**Comment by rabernat** [comment]
Hi @SimonKrughoff, thanks for reaching out! I hope you were able to resolve your problem.

Let us know if you want to chat more generally about adopting the pangeo approach to astronomy. I think there are great potential synergies. The more multi-platform we can make these tools, the more sustainable they will be.

**Comment by SimonKrughoff** [comment]
@mrocklin I am still seeing the problem. A coworker was able to reproduce the behavior. Perhaps it's my network, but I tried over a VPN and got the same thing. I'll try again from home this evening. Here's a screenshot of what I see.
<img width="991" alt="image" src="https://user-images.githubusercontent.com/945715/40022494-0c5f06e8-577d-11e8-8770-4ee3342609b1.png">

@rabernat this ecosystem certainly appears to be emerging as a very powerful environment. I think we should definitely keep lines of communication open. I've joined your gitter channel.
